cmd/crd-example/allall.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: aci.aw/v1
```

Comment:
What does .aw stand for?

Reply:
AW as in AnyWhere. But, we could change this as appropriate later.